src/Apps/Order/Routes/Shipping2/Components/__tests__/FulfillmentDetailsForm.jest.tsx

```diff
@@ -187,11 +185,9 @@ describe("FulfillmentDetailsForm", () => {
       await flushPromiseQueue()
 
       await waitFor(() => {
-        ;["Full name is required", "Phone number is required"].forEach(
-          error => {
-            expect(screen.getByText(error)).toBeInTheDocument()
-          }
-        )
+        ;["Phone number is required"].forEach(error => {
+          expect(screen.getByText(error)).toBeInTheDocument()
+        })
```

Comment:
**Suggestion**: maybe we could update it to one instead of an array?
```suggestion
        expect(screen.getByText("Phone number is required")).toBeInTheDocument()
```